src/jmh/java/net/sf/jabref/benchmarks/Benchmarks.java

```diff
@@ -0,0 +1,91 @@
+package net.sf.jabref.benchmarks;
+
+import java.io.IOException;
+import java.io.StringReader;
+import java.io.StringWriter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+
+import net.sf.jabref.*;
+import net.sf.jabref.exporter.BibDatabaseWriter;
+import net.sf.jabref.exporter.SaveException;
+import net.sf.jabref.exporter.SavePreferences;
+import net.sf.jabref.importer.ParserResult;
+import net.sf.jabref.importer.fileformat.BibtexParser;
+import net.sf.jabref.logic.search.SearchQuery;
+import net.sf.jabref.model.database.BibDatabase;
+import net.sf.jabref.model.database.BibDatabaseMode;
+import net.sf.jabref.model.database.BibDatabaseModeDetection;
+import net.sf.jabref.model.entry.BibEntry;
+import org.openjdk.jmh.Main;
+import org.openjdk.jmh.annotations.*;
+import org.openjdk.jmh.runner.RunnerException;
+
+@State(Scope.Thread)
+public class Benchmarks {
```

Comment:
Not that I'm aware of. In the end JMH only runs the benchmarks and writes a result file.

I found plugins for [jenkins](https://github.com/blackboard/jmh-jenkins) and for [teamcity](https://github.com/presidentio/teamcity-plugin-jmh) but apperently there is nothing similar for circleci or travis.